agent/fw_drupal8.c

```diff
@@ -605,10 +605,19 @@ NR_PHP_WRAPPER_END
 #endif  // OAPI
 
 static bool nr_is_invalid_key_val_arr(nr_php_string_hash_key_t* key,
-                                      zval* val) {
+                                      zval* val,
+                                      const char* key_ident) {
   if (NULL == key || 0 == ZEND_STRING_LEN(key)
       || 0 == nr_php_is_zval_valid_array(val)
       || 0 == zend_hash_num_elements(Z_ARRVAL_P(val))) {
+    if (NULL != key) {
+      nrl_warning(NRL_FRAMEWORK,
+                  "hookImplementationsMap[%s]: invalid value for key '%s'",
+                  key_ident, ZEND_STRING_VALUE(key));
```

Comment:
enclose both key_idents in `NRSAFESTR` just in case

Reply:
I don't think `NRSAFESTR` is necessary here- `key_ident` is not derived, it is a manually defined and passed `const char *` passed explicitly by the developer